### ISSUE TITLE
Enable renaming categories

### ISSUE THREAD

**Issue by sircode** [nextcloud/notes-android#2434]
### Is your feature request related to a problem? Please describe.

Rename categories

### Describe the solution you'd like

Rename categories

**Comment by stefan-niedermann** [comment]
Whats wrong with the <kbd>Category</kbd> menu item in the three dots menu of each note to change the category?

**Comment by sircode** [comment]
Yes, one can change the category, but one can no rename  categories. At least I couldn't find that feature anywhere

**Comment by Kokika** [comment]
You can rename the folder which is the category name in Nextcloud files, however it can not be done in Note app.
